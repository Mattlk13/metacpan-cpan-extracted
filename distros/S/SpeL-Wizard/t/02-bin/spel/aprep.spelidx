format|ogg
audiodir|spel
chunkdir|spel
macpp|testmacro|3|default value|dit is ##1 en dit is ##2 en ten slotte ##3!
language|english
chunk|chunk-aprep
