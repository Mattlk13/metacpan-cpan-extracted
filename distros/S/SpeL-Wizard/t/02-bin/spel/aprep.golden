Ik wil de macropreprocessor testen: jawel dit is optarg en dit is mandarg with-nesting en ten slotte 2ndmandarg! overflow en nog wat trailing dit is default value en dit is 2ndtest en ten slotte anothermandarg! nog een overflow en wat trailing.
